recipes/Text_to_Shell/Text_to_Shell.ipynb

```diff
@@ -91,31 +214,79 @@
     "Question:\n",
     "create a tar ball of all pdf files in current folder\n",
     "Answer:\n",
-    "find . -name *.pdf | xargs tar czvf /root/Desktop/evidence/pdf.tar\n",
+    "find . -name *.pdf | xargs tar czvf pdf.tar\n",
     "\n",
     "Question:\n",
-    "Sort all files/directories under current directory according to modification time and print only the recent 7 of them\n",
+    "Sort all files/directories in the current directory, but no subdirectories, according to modification time and print only the recent 7 of them\n",
     "Answer:\n",
-    "find -mindepth 1 -printf \\\"%T@ %Pn\\\" | sort -n -r | cut -d' ' -f 2- | tail -n +7\n",
+    "find . -maxdepth 1 -exec stat -f \"%Sm {}\" \\; | sort -n -r | tail -n 7\n",
     "\n",
     "Question:\n",
     "find all the empty directories in the current folder\n",
     "Answer:\n",
     "find . -type d -empty\n",
     "\n",
-    "\"\"\"\n",
-    "\n",
-    "query = \"How can I use a bash script to calculate the number of days between the dates Jan 12, 2023 and Jan 20, 2024?\"\n",
-    "\n",
-    "user_prompt = examples + \"\\nQuestion:\\n\" + query + \"\\nAnswer:\\n\"\n",
-    "\n",
-    "response = ollama.chat(model='granite-code:8b', messages=[\n",
-    "  {\n",
-    "    'role': 'user',\n",
-    "    'content': user_prompt\n",
-    "  },\n",
-    "])\n",
-    "print(response['message']['content'])"
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's define another helper function for calling `ollama.chat()`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def chat1(prompt: str, examples: str = examples, model: str ='granite-code:8b') -> str:\n",
+    "    user_prompt = f\"\"\"\n",
+    "        {examples}\n",
+    "        Question:\n",
+    "        {prompt}. Make sure you generate code that is {os_name()}-compatible!\n",
+    "        Answer:\"\"\"\n",
+    "    response = ollama.chat(model=model, messages=[\n",
```

Comment:
Do we need to use the `chat` function for this? I may have started with that, but I think we may want to save `chat()` for when there are multiple exchanges.

As above, let's use the system prompt for general meta-instructions, such as "Make sure you generate code that is {os_name()}-compatible!".